### PR TITLE
[skip ci] [重大更新] Support QQ 8.5.0 & Add RemoveIntimateDrawer.kt

### PR DESCRIPTION
**Pull request by NextAlone**
# Support QQ 8.5.0

### Initially support QQ 8.5.0

- Add RemoveIntimateDrawer.kt
- Update RemovePlayTogether.java, CheatHook.java, SimplifyQQSettingMe.kt, ForceSystemCamera.kt(only video), RemoveGroupApp.kt, BaseDelayableHook.java, DexKit.java, pref_chat.xml.
- Fixed FaceImpl.java(which causes FC). 
- Deprecated Utils.getBusinessHandler().
- Reconstructed Utils.getFriendListHandler().
### TODO
- Fixed New Round Avatar.

## 更改内容 / Types of Changes

- [x] 新功能 New feature
- [x] 修复 Bugfix

## 检查列表 / Checklist

- [x] 我的代码符合本仓库的代码规范 My code follows the code style of this project
- [x] 我已经测试了这些更改，它们可以正常工作，且不会破坏任何功能(或我可以解决) I have tested the changes and verified that they work and don't break anything (as well as I can manage).
- [x] 我已合并了对后续工作无意义的commit并确认不会对后续维护造成困扰 I have merged commits that are meaningless for follow-up work and confirmed that they will not cause damage to follow-up maintenance

### CODE REVIEW

**Review by Lagrio — APPROVED** [review]

**Review by cinit — CHANGES_REQUESTED** [review]
`getBusinessHandler()` 存在问题

File: app/src/main/java/nil/nadph/qnotified/util/Utils.java [hunk 1]
```diff
@@ -1220,12 +1238,18 @@ public static Object getBusinessHandler(int type) {
                 return invoke_virtual(appInterface, "a", type, int.class, cl_bh);
             } catch (NoSuchMethodException e) {
                 try {
-                    Method m = appInterface.getClass().getMethod("getBusinessHandler", int.class);
-                    m.setAccessible(true);
-                    return m.invoke(appInterface, type);
+                    return invoke_virtual(appInterface, "getBusinessHandler", type, String.class,
```
[the remark this answers is not in the document]
Comment (NextAlone) [reply app/src/main/java/nil/nadph/qnotified/util/Utils.java:1241]: 改成...return getBusinessHandler("com/tencent/mobileqq/app/FriendListHandler");

public static Object getBusinessHandler(String type) {
...
return invoke_virtual(appInterface, "getBusinessHandler", type, String.class, cl_bh);
...
}

返回了空指针

[the remark this answers is not in the document]
Comment (cinit) [reply app/src/main/java/nil/nadph/qnotified/util/Utils.java:1241]: 我们可以删除方法 `Utils.getBusinessHandler(任何参数?)` , 因为唯一会用到这个方法的地方是 `Utils.getFriendListHandler()` , 我们只要把 `Utils.getBusinessHandler(?)` 丢到 `Utils.getFriendListHandler()` 里重新实现一下, 然后就可以删除 `Utils.getBusinessHandler(?)` 了

[the remark this answers is not in the document]
Comment (NextAlone) [reply app/src/main/java/nil/nadph/qnotified/util/Utils.java]: 已弃用getBusinessHandler()，且无Exception了

**Review by cinit — APPROVED** [review]

**Review by JasonKhew96 — APPROVED** [review]

**Pull request merged**